### PR TITLE
Rails Budget

### PR DESCRIPTION
**Pull request by jasonfelice**
# Rails Budget App
>This Rails Budget app allows users to record their transactions for a specific category.

## Presentation [video](https://youtu.be/sMet8FA7KAg). Here's a list of tasks I've done in this PR:
- [x] Set up database
- [x] Generated models and controllers
- [x] Set up Bootstrap and flash messages
- [x] Created and styled the following pages:
  - Splash
  - Login/Sign Up
  - Category index & new
  - Record index & new

### CODE REVIEW

File: app/controllers/records_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class RecordsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @category = current_user.categories.find(params[:category_id])
+    @records = @category.records
+  end
```
Comment (DammyShittu) [inline app/controllers/records_controller.rb]: - [ ] Your transactions on the page should be displayed based on the most recent record. So for example if I have 2 records (record 1 and record 2), the page should display the records in this order: `record 2`, `record 1` based on the most recent transaction.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Budget App
-> Budget App
```
Comment (DammyShittu) [inline README.md:2]: - [ ] Good job adding the unit tests for the models. However, you are still missing integration tests for the application. Kindly implement those also.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @jasonfelice,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
H @jasonfelice, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**